cmd/zoekt-sourcegraph-indexserver/queue.go

```diff
@@ -176,12 +185,14 @@ func (q *Queue) handleDebugQueue(w http.ResponseWriter, r *http.Request) {
 
 	writer := tabwriter.NewWriter(&bufferedWriter, 16, 8, 4, ' ', 0)
 
-	_, err := fmt.Fprintf(writer, "Position\tName\tID\tIsOnQueue\tBranches\t\n")
+	_, err := fmt.Fprintf(writer, "Position\tName\tID\tIsOnQueue\tTimeSpentAwaitingIndex\tBranches\t\n")
```

Comment:
minor non-blocking: quote a verbose name. Not sure if I have a more succinct suggestion. Maybe age?

Reply:
I'll try switching to $AGE here, but I'll also add a longer description for each column in the debug command's help output.